### DISCUSSION TITLE
Which term will we use  topic or tag for notifications?

### DISCUSSION THREAD

**Discussion by bgmrsln** [bounswe/bounswe2023group2#96] [Q&A]
<img width="958" alt="image" src="https://user-images.githubusercontent.com/53114791/230721405-f89346c9-e985-4e5d-a8fc-c0e2bd4eed62.png">
I write some requirements related to notifications.

**Comment by bgmrsln** [comment]
User req

1.1.3.5 Users should be able to receive notifications about available resources, or actions taken in their area.(at what level) -> 1.1.3.5. Users should be able to receive notifications about certain topics based on location.(topic will be explained on glossary)
1.2.3.6 Users shall be able to search a resource, event or action, and receive results semantically related to the search (already exists)
1.2.3.6.1. Users should be able to subscribe to a topic based on thier filter and search
System Req

1.2.10.2: The system should be able to create notifications, if they click “want to be notified” button on information. -> 1.2.10.2: The system should create notifications, if they click “want to be notified” button on information filter and search.
1.2.10.3: The system should be able to create notifications if users' profession might be implying that they can be a resource to a certain need. (exists)
1.2.10.3.1: system should be able to add resource | translator | any topic to notification settings as default when user have a role
1.2.10.4: The system should be able to create notifications if an event takes place near users' addresses when they provide their addresses. (exists)
1.2.10.4.1 system should be able to add event | any | (user's city) topic to notification settings as default.
will add topic to glossary: (activity | type | location) ex: need | food | Istanbul

**Comment by rburaksaritas** [comment]
Topic seems okay imo.

**Comment by bgmrsln** [comment]
Will continue with topic
